CHANGES.rst

```diff
@@ -1,6 +1,13 @@
 Changes
 =======
 
+Version 0.4.0 (2018-11-06)
+--------------------------
+
+- Improves AMQP re-connection handling. Switches from ``pika`` to ``kombu``.
```

Comment:
Actually we still have `pika` in the requirements... Could be deleted here and elsewhere...